admin/inc/functions_themes.php

```diff
@@ -502,24 +499,6 @@ function resync_stylesheet($stylesheet)
 	return false;
 }
 
-/**
```

Comment:
I thought you accidentally included this. Removing functions in a minor point release may introduce compatibility issues.